docs/Common-Issues.md

```diff
@@ -113,6 +113,90 @@ xcode-select --install
 brew upgrade
 ```
 
+## Cask - cURL error
```

Comment:
Would it break the structure of this document too much if we make `h2` headings for "Formula Issues", "Cask Issues", "Other Issues" (or something like that) and then have everything else shifted down a heading level (i.e. this would become `h3` and its sub-sections `h4`) and a table of contents at the very top so if you know what category your problem falls under, you can easily navigate there?

Reply:
I think these are the only issues that are specific to a tap, so that's why I added them like this. It might even be better in the troubleshooting guide (although I'm not sure what the difference is supposed to be)

Reply:
👍 Gotcha, makes sense to leave as-is, then.

It looks like the Troubleshooting page is really about how to report issues rather than how to solve them on your end, but I can see why that's confusing. Maybe we should rename it to "Reporting Issues" or something similar so the go-to document for people having trouble with anything is "Common Issues" (we could also have a link at the bottom of that page saying "if nothing works you can try reporting by following these instructions" and linking